BibBuddy/app/src/main/java/de/bibbuddy/NoteModel.java

```diff
@@ -75,6 +75,25 @@ public Note getLastNote() {
     return createItemList(noteList);
   }
 
+  /**
+   * This method gets the list of all existing notes of the voice-type (int=1).
+   *
+   * @return returns the list of voice type note objects.
+   */
+  public List<Note> getVoiceNoteList() {
+    List<Note> noteList = noteDao.findAll();
+    for (int i = 0; i < noteList.size(); i++) {
+      if (noteList.get(i).getType() != 1) {
+        noteList.remove(noteList.get(i));
+      }
+    }
+    return noteList;
+  }
```

Comment:
Would add newlines here, may be subjective.

```suggestion
  public List<Note> getVoiceNoteList() {
    List<Note> noteList = noteDao.findAll();
    
    for (int i = 0; i < noteList.size(); i++) {
      if (noteList.get(i).getType() != 1) {
        noteList.remove(noteList.get(i));
      }
    }
    
    return noteList;
  }
```